docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst

```diff
@@ -0,0 +1,43 @@
+##################
+Tutorial Notebooks
+##################
+
+.. _NB-Intro-Use-Tutorial-NBs:
+
+How to use the Tutorial Notebooks
+=================================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the Rubin/LSST tutorial notebooks which are provided in each user's home directory.
+
+**The "notebooks/tutorial-notebooks" directory is read-only:**
+The read-only "notebooks/tutorial-notebooks" directory will *always* contain the most up-to-date versions of the tutorials.
+Notebooks can be edited and executed in this directory, but **changes cannot be saved to this directory**.
+Users wishing to edit, execute, *and save* versions of these notebooks should copy them to a different path in their home directory.
+
+**How to obtain an editable version of a tutorial notebook:**
+The commands below demonstrate how to create a copy of a tutorial notebook in the home directory which can be opened, edited, and saved.
+Step-by-step, the commands below change directory (``cd``) into the home directory (``~``),
+copy the desired tutorial to the (``cp``) into the current directory (``.``),
+list (``ls``) the files that are in the current directory to confirm the copy worked,
+and list in long format all attributes in human-readable form (``-lah``) for the copied file.
+The standard output ``-r--r--r--`` indicates that the file is read-only (``r``) by the user, the group, and everyone
+with access to the file (the three ``r``).
+Change the mode (``chmod``) of the file to add user write access (``u+w``), and repeate the
```

Comment:
"repeate" --> "repeat"

Reply:
Good catch, fixed in [d7b6135](https://github.com/lsst/rsp_lsst_io/commit/d7b6135d513e384b89f2195aec734fa8cb3d14f8), thanks